docs/checksum32.rst

```diff
@@ -13,6 +13,17 @@ CRC32
     .. automethod:: from_config
 
 
+CRC32C
+------
+.. autoclass:: CRC32C
```

Comment:
😱 I forgot we were still doing things manually in the docs, this is a good reminder for me to fix this in another PR  😄